src/components/google-maps-aerial.js

```diff
@@ -0,0 +1,144 @@
+import { TilesRenderer } from '3d-tiles-renderer';
+import {
+  TilesFadePlugin,
+  TileCompressionPlugin,
+  GLTFExtensionsPlugin,
+  GoogleCloudAuthPlugin
+} from '3d-tiles-renderer/plugins';
+
+console.log('3d-tiles-renderer', TilesRenderer);
+const MathUtils = AFRAME.THREE.MathUtils;
+const DRACOLoader = AFRAME.THREE.DRACOLoader;
+
+if (typeof AFRAME === 'undefined') {
+  throw new Error(
+    'Component attempted to register before AFRAME was available.'
+  );
+}
+
+AFRAME.registerComponent('google-maps-aerial', {
+  schema: {
+    apiToken: { type: 'string', default: '' },
+    latitude: { type: 'number', default: 37.795 }, // SF Ferry Building
+    longitude: { type: 'number', default: -122.394 },
+    minDistance: { type: 'number', default: 500 },
+    maxDistance: { type: 'number', default: 20000 },
+    ellipsoidalHeight: { type: 'number', default: 0 },
+    copyrightEl: { type: 'selector' }
+  },
+
+  init: function () {
+    this.tick = AFRAME.utils.throttleTick(this.tick, 10, this);
```

Comment:
It doesn't make sense. 10 is lower than 16ms. Did you mean 100ms?

Reply:
Instead of tick, did you try `Render On Change`?
 https://github.com/NASA-AMMOS/3DTilesRendererJS/tree/254774c7e0d3849a4a964633ed6376b5fbc2414c?tab=readme-ov-file#render-on-change

Reply:
you're right the tick was useless bc3c45efbceec4037081297d4ae1cba4dc4fbf7b
i'll take a look at render on change

Reply:
I tried render on change and it wasn't working as expected. i'm going to leave as is for now, the most expensive operation to fix next is applying blending and clipping, the tiles update on each tick is not worse than before

The issue: load-model event only seems to fire when running tiles.update() in each tick, otherwise that event wouldn't fire.